install/sles-installation-guide.md

```diff
@@ -0,0 +1,47 @@
+# Installing Kata Containers on SLES
+
+Note: Kata Containers is available for SLES version 12 SP3.
+
+This step is only required in case Docker is not installed on the system.
+1. Install the latest version of Docker with the following commands:
+
+```bash
+$ sudo zypper -n install libcgroup1
+$ curl -OkL https://yum.dockerproject.org/repo/main/opensuse/13.2/Packages/docker-engine-17.05.0.ce-1.x86_64.rpm
```

Comment:
I think we should remove the steps of adding this repo and install docker-engine. This docker version is 17.05, which is older than the one that SUSE provides, which is 17.04